docs/UserGuide.md

```diff
@@ -1,21 +1,21 @@
+![AthletiCLI Banner with fitness imagery](images/AthletiCLI-Banner.png)
```

Comment:
To set the feature image, we can set the `feature_image` in the YAML block.
(See docs/README.md for example.)

Reply:
If we need to add pictures, it is best to add them to other pages as well.